contents/dart/Dart.md

```diff
@@ -0,0 +1,228 @@
+<frontmatter>
+  title: Introduction to Dart
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Dart
+**Author(s): [Tan Yuanhong](https://github.com/le0tan)**<br>
+
+## What Is Dart?
+Dart (previously also known as `dartlang`) is an object-oriented, <tooltip content="a style of Object-oriented programming (OOP) in which inheritance occurs via defining classes of objects, instead of inheritance occurring via the objects alone (compare prototype-based programming)">class defined</tooltip>, garbage-collected language using a C-style syntax that <tooltip content="also known as source-to-source compilation">transcompiles</tooltip> optionally into JavaScript. It's rumored that Dart was invented out of Google engineers' frustration with JavaScript (they even built [a version of Chromium browser](http://dartdoc.takyam.com/tools/dartium/) with Dart VM so that Dart code can be run on the web without transcompiling to JavaScript). However, as most developers still stick to JavaScript and it turned out that TypeScript is a much more widely-accepted solution for statically typed JavaScript transcompilation, Dart is then, with the emergence of <tooltip content="a cross-platform mobile UI framework developed by Google">Flutter</tooltip>, re-purposed as a **client-optimized** language that's optimized for **UI creation** and **cross-platform execution**.
+
+## Why Learn Dart?
+### Flutter
+**If you are to try Flutter, you have to learn Dart. :smirk:** Let's face it, Dart is basically <tooltip content="at least to most developers, Dart has never been dead inside Google as Google's most critical business - Google Ads is written entirely with Dart.">"revived"</tooltip> by the development of Flutter, and that's not a bad thing at all - there are many excellent programming languages that don't receive much attention simply because they're not used in production by huge companies or by an active community. In other words, as long as Google is still committed to maintaining Flutter and the community is still building apps using Flutter, Dart remains an actively updated programming language.
```

Comment:
Try to sell this instead to the perspective of someone who is new to both flutter and dart, while keeping it simple ( e.g. remove the extra information on `...revived...` `... google is committed...` and instead talk a little more about flutter )